src/krate.rs

```diff
@@ -399,19 +402,33 @@ impl Crate {
         parts.next().is_none()
     }
 
-    pub fn minimal_encodable(self,
-                             max_version: semver::Version,
-                             badges: Option<Vec<Badge>>, exact_match: bool) -> EncodableCrate {
-        self.encodable(max_version, None, None, None, badges, exact_match)
+    fn minimal_encodable(self,
+                         max_version: semver::Version,
+                         badges: Option<Vec<Badge>>,
+                         exact_match: bool,
+                         max_build_info: Option<EncodableMaxBuildInfo>)
+                         -> EncodableCrate
+    {
+        self.encodable(
+            max_version,
+            None,
+            None,
+            None,
+            badges,
+            exact_match,
+            max_build_info,
+        )
     }
 
+    #[cfg_attr(feature = "lint", allow(too_many_arguments))]
```

Comment:
I do think we should refactor this, I am agnostic about whether it should be done in this PR or in a separate PR